sdk/digitaltwins/Azure.DigitalTwins.Core/src/Generated/QueryRestClient.cs

```diff
@@ -16,8 +16,8 @@ namespace Azure.DigitalTwins.Core
 {
     internal partial class QueryRestClient
     {
-        private Uri endpoint;
```

Comment:
Isn't this generated code? Why are you changing it?

Reply:
Good point. I lost my context. :)